Lib/logging/__init__.py

```diff
@@ -800,17 +800,14 @@ def filter(self, record):
         .. versionchanged:: 3.2
 
            Allow filters to be just callables.
+
+        .. versionchanged: 3.9
+
+           Improve readability using efficient Python 3 builtins.
```

Comment:
This note is unnecessary; there's no change to the behavior, so there's no reason for the user to care.  Also, the version targeted by this change is currently 3.8.

Reply:
Oops. Fixed. I'm assuming if this ever gets merged, the various commits are squashed together.

Reply:
Oh now I see what you mean.... pushing a new commit without the note.